spec/lib/consul_form_builder_spec.rb

```diff
@@ -75,6 +75,15 @@
     end
   end
 
+  describe "#check_box" do
+    it "renders the label with new checkbox-label class by default" do
```

Comment:
I think there's a typo here :thinking:. An article is missing and the class won't be "new" when we read the test in a few months :smile:.

```suggestion
    it "adds a checkbox-label class to the label by default" do
```